pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -73,7 +73,7 @@ type EventListenerTrigger struct {
 // EventInterceptor provides a hook to intercept and pre-process events
 type EventInterceptor struct {
 	Webhook *WebhookInterceptor `json:"webhook,omitempty"`
-	Github  *GithubInterceptor  `json:"github,omitempty"`
+	GitHub  *GitHubInterceptor  `json:"github,omitempty"`
```

Comment:
Q: should this be `gitHub`? I don't think so, but I'd like feedback.

Reply:
@ImJasonH doesn't it need to be exported to be successfully marshalled ? (and thus `gitHub` wouldn't work)

Reply:
Are you talking about the json struct tag being `gitHub` instead of `github`? I think `github` is fine

Reply:
oh the json struct :sweat_smile:

Reply:
Yeah the json struct tag. I agree `github` is fine.